Source/SwiftLintFramework/Rules/ExplicitAssociatedEnumValueRule.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ExplicitAssociatedEnumValueRule.swift
+//  SwiftLint
+//
+//  Created by Mazyad Alabduljaleel on 8/19/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+public struct ExplicitAssociatedEnumValueRule: ASTRule, OptInRule, ConfigurationProviderRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "explicit_associated_enum_value",
```

Comment:
Just for future reference: https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Enumerations.html#//apple_ref/doc/uid/TP40014097-CH12-ID148